EGO Forum Enhancement.user.js

```diff
@@ -559,6 +645,20 @@ function handleApplicationPage() {
     else if (url.match(/^https:\/\/www\.edgegamers\.com\/application\/\d+\/?$/))
         // Application Page
         handleApplicationPage();
+    else if (url.match(/^https:\/\/www\.edgegamers\.com\/award-system\/user\/awards\?user=\d+\/?$/))
+        // User Award Page
+        handleUserAwardPage();
+    else if (url.match(/^https:\/\/www\.edgegamers\.com\/award-system\/\d+\/recent\/?$/))
+        // Recent Award Page
+        handleRecentAwardPage();
+    else if (url.match(/^https:\/\/www\.edgegamers\.com\/award-system\/\d+\/recent\?username=.*$/)) {
```

Comment:
Your Regex here would match an empty/non-existent username parameter, which isn't ideal.

```suggestion
    else if (url.match(/^https:\/\/www\.edgegamers\.com\/award-system\/\d+\/recent\?username=.+$/)) {
```